data/summary/types/cgm.go

```diff
@@ -35,6 +35,9 @@ type CGMBucketData struct {
 	TotalGlucose float64 `json:"totalGlucose" bson:"totalGlucose"`
 	TotalMinutes int     `json:"totalMinutes" bson:"totalMinutes"`
 	TotalRecords int     `json:"totalRecords" bson:"totalRecords"`
+
+	RealtimeRecords int `json:"realtimeRecords" bson:"realtimeRecords"`
```

Comment:
I think you should leave a comment here describing what's a realtime record and what's a deferred record.

Reply:
Good idea, added.